core-services/prow/02_config/openshift/cluster-samples-operator/_pluginconfig.yaml

```diff
@@ -1,3 +1,10 @@
+label:
+  restricted_labels:
+    openshift/cluster-samples-operator:
+    - allowed_users:
```

Comment:
I *think* you should have an 
``
  allowed_teams:
  - openshift-patch-managers
 ```
entry  preceding the `allowed_users` entry in the `openshift/cluster-samples-operator` array

I at least see that @dperaza4dustbit in the components @adambkaplan and @coreydaley set up for build-api team